applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -785,6 +786,7 @@ int le_audio_enable(le_audio_receive_cb recv_cb)
 {
 	int ret;
 
+	bt_conn_cb_register(&conn_callbacks);
```

Comment:
Probably we move this line into initialize(), since all the registrations are happened in there.

Reply:
Move to initialize().